### DISCUSSION TITLE
ENV variable to set Unit processes max and spare

### DISCUSSION THREAD

**Discussion by miscs** [shinsenter/php#149] [Q&A]
Hi,

thank you for all the hard work! 
For FPM I understand how to set PM_MAX_CHILDREN and other related settings by ENV. But how to set processes.max and processes.spare for Nginx Unit server? Is there also a ENV for this?

Thanks!

**Comment by shinsenter** [comment]
Hi @miscs,

Thank you for your question.

The Docker images I provide for all variants mainly focus on simplifying the configuration for PHP and PHP-FPM. As for the included middleware (such as Nginx Unit), most of them use basic configurations.

To update the configurations according to your needs, you can refer to my sample JSON file below, make the necessary adjustments, and then copy or mount that JSON file into the `/etc/unit/sites-enabled` directory of the container. When the container starts, it will read the JSON files in `/etc/unit/sites-enabled` and load them into Nginx Unit.

https://github.com/shinsenter/php/blob/afec8b85e82035cc651e186598a44bb4f4a6695b/src/php/with-unit/rootfs/etc/unit/sites-enabled/00-default.json

I hope this information helps!
